_content/booklets/what-is-kamma_shravasti-dhammika.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Good Kamma! Bad Kamma! What Exactly is Kamma?"
+authors:
+  - "Shravasti Dhammika"
+external_url: "https://wiswo.org/books/_resources/book-downloads/gkbk/gkbk.pdf"
```

Comment:
Bhante has asked that we use the latest URL instead: https://budblooms.org/download/good-kamma-bad-kamma-what-exactly-is-kamma/